bitnami/prometheus-operator/templates/prometheus-operator/deployment.yaml

```diff
@@ -100,4 +111,14 @@ spec:
           securityContext:
             allowPrivilegeEscalation: false
             readOnlyRootFilesystem: false
+---
```

Comment:
We usually create a "configmap.yaml"/"cm.yaml" file to make it easier to maintain later on. Could you add it?